docs/_sass/_colors.scss

```diff
@@ -1,17 +1,8 @@
-$color-dark: #0B1B34;
-$color-gray1: #FAFAFA;
-$color-gray2: #ECECEC;
-$color-gray3: #C6C9CA;
-$color-green: #07d973;
-$color-pink: #F68DFE;
-$color-white: #FFFFFF;
-
-$color-light-grey-green: #C9D3C5;
-$color-gray-green: #8B9C8F;
-$color-dark-green: #1B5744;
-$color-darker-green: #002E22;
-$color-super-dark-green: #061B09;
-$color-light-blue: #8DC8FF;
-
-$color-link: #5AB0FF;
-$color-link-hovered: #B0D9FF;
+$color-green400: #03D47C;
+$color-white: #E7ECE9;
+$color-green-icons: #8B9C8F;
+$color-green-borders: #1A3D32;
+$color-green-highlightBG: #07271F;
+$color-green-appBG: #061B09;
+$color-blue200: #5AB0FF;
```

Comment:
Per our discussion, this should be blue300

Reply:
@shawnborton I removed old blue200 because it is used  to be set color of a tag but a tag that have text in children is apply this blue200 or color-success and old blue200 doesn't use in any other